src/App.tsx

```diff
@@ -1,9 +1,22 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchMovie, setSearchMovie] = useState('');
+
+  const isMovieIncludeQuery = (value: string) => Boolean(
```

Comment:
Looks like boolean variable but it's function

Reply:
renamed